src/evaluations/evaluator.py

```diff
@@ -224,38 +237,94 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
     self.scenario_random_states = scenario_random_states
 
   def __call__(self):
-    self.evaluate_all()
-
-  def evaluate_all(self):
-    """Evaluate all estimators under all scenarios."""
+    self.evaluate_all_parallel()
+
+  def evaluate_all_parallel(self):
```

Comment:
Can you consider breaking this function up into multiple smaller methods or at least moving the very long nested method out to a private module level function?

Reply:
The nested method has been moved into a private class-level function in 7ea9926.